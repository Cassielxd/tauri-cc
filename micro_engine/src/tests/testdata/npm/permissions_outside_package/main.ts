import { loadConfigFile } from "npm:@denotest/permissions-outside-package";

const fileName = `${Deno.cwd()}/npm/permissions_outside_package/foo/config.js`;
const config = loadConfigFile(fileName);
console.log(config);
