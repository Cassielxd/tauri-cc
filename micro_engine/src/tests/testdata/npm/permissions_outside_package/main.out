Download http://localhost:4545/npm/registry/@denotest/permissions-outside-package
Download http://localhost:4545/npm/registry/@denotest/permissions-outside-package/1.0.0.tgz
{ name: "foobar", version: "0.0.1" }
